Spec/Auth.swift

```diff
@@ -1903,6 +1903,32 @@ class Auth : QuickSpec {
                         done()
                     }
                 }
+                
+                tokenParams.clientId = "testId"
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.authorise(tokenParams, options: authOptions) { tokenDetails, error in
+                        expect(error).to(beNil())
+                        guard let tokenDetails = tokenDetails else {
+                            XCTFail("TokenDetails is nil"); done(); return
+                        }
+
+                        expect(tokenDetails.clientId).to(equal("testId"))
+                        done()
+                    }
+                }
+                
+                tokenParams.clientId = nil
+                waitUntil(timeout: testTimeout) { done in
+                    rest.auth.authorise(tokenParams, options: authOptions) { tokenDetails, error in
```

Comment:
> This has the same issue as #491

Got it, thanks.

> Also the argument is incorrect.

As per our recent conversation in #491 - we agreed that `createTokenRequest` API has correct set of arguments. 
Can we assume that this also correct for `authorise` method [signature](https://github.com/ably/ably-ios/blob/master/Source/ARTAuth.h#L47)?